client/blocks/product-purchase-features-list/custom-css.jsx

```diff
@@ -0,0 +1,51 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import PurchaseDetail from 'components/purchase-detail';
+import { isEnabled } from 'config';
+
+/**
+ * Image dependencies
+ */
+import customizeImage from 'assets/images/illustrations/dashboard.svg';
+
+function isCustomizeEnabled() {
+	return isEnabled( 'manage/customize' );
+}
+
+function getEditCSSLink( selectedSite ) {
+	const adminUrl = selectedSite.URL + '/wp-admin/',
+		customizerInAdmin =
+			adminUrl +
+			'customize.php?return=' +
+			encodeURIComponent( window.location.href ) +
+			'&section=jetpack_custom_css';
+
+	return isCustomizeEnabled() ? '/customize/custom-css/' + selectedSite.slug : customizerInAdmin;
+}
+
+export default localize( ( { selectedSite, translate } ) => {
+	return (
+		<div className="product-purchase-features-list__item">
+			<PurchaseDetail
+				icon={ <img alt="" src={ customizeImage } /> }
+				title={ translate( 'Custom CSS' ) }
+				description={ translate(
+					"Make advanced changes to your site's appearance by " + 'writing your own CSS.'
```

Comment:
What is this string concat for? The initial double quotes are obviously to avoid escaping the 's, but why is the last bit appended in single quotes? Seems like enclosing the full string in double quotes would be easier, but maybe there is some translate magic here that I don't know about :-)

Reply:
+1 on this

Reply:
No reason, other than the advanced css card did the same thing, I will remove this.